src/js/util/AuthService.jsx

```diff
@@ -11,8 +11,10 @@ var url = "https://dev.calligre.com"
 class AuthService extends EventEmitter {
   constructor(clientId, domain) {
     super()
+    this.clientId = clientId
+    this.domain = domain
     // Configure Auth0
-    this.lock = new Auth0Lock(clientId, domain, {auth: {redirect: false}})
+    this.lock = new Auth0Lock(clientId, domain, {auth: {redirect: true}})
```

Comment:
Since we're redirecting you should remove the on hide listener below (git wont let me comment on the line, but its lines 25-27)